src/taglibs/migrate/async-fragment-tag.js

```diff
@@ -0,0 +1,59 @@
+const migrateControlFlowDirectives = require("./control-flow-directives");
+const newTags = {
+    "async-fragment": "await",
+    "async-fragments": "await-reorderer",
+    "async-fragment-placeholder": "await-placeholder",
+    "async-fragment-timeout": "await-timeout",
+    "async-fragment-error": "await-error"
+};
+module.exports = function migrator(oldNode, context) {
+    const oldTag = oldNode.tagName;
+    const newTag = newTags[oldTag];
+    var provider;
+    var varName;
+    var argument;
+    const attributes = oldNode.attributes;
+    const builder = context.builder;
+    migrateControlFlowDirectives(oldNode, context);
```

Comment:
The var/assign/invoke tags needed to do this because they were being migrated to a scriptlet which is not an HtmlElement and doesn't have attributes, so the control flow would have been lost if it wasn't handled here.

```js
migrateControlFlowDirectives(oldNode, context);
oldNode.setTransformerApplied(migrateControlFlowDirectives);
```

However, I think these lines aren't necessary in this case because the attributes get copied over to the new element if the migration hasn't already run.